File: tests/assets/src/java_best_practices_present/ControlsPresent.java
import com.google.android.gms.safetynet.SafetyNetApi;

import android.view.View;
import android.view.WindowManager;

import okhttp3.CertificatePinner;

class ControlsPresent extends Activity {
    void enableControls(View view, Device device) {
        getWindow().addFlags(WindowManager.LayoutParams.FLAG_SECURE);
        device.isDeviceRooted();
        view.setFilterTouchesWhenObscured(true);
        new CTInterceptorBuilder();
        new CertificatePinner.Builder();
    }
}

File: tests/assets/src/kotlin_best_practices_present/ControlsPresent.kt

import android.view.WindowManager
import com.google.android.gms.safetynet.SafetyNet
import okhttp3.CertificatePinner

fun enableControls(window: android.view.Window, view: android.view.View, device: Device) {
    SafetyNet.getClient(context).attest(nonce, apiKey)
    window.addFlags(WindowManager.LayoutParams.FLAG_SECURE)
    device.isDeviceRooted()
    view.setFilterTouchesWhenObscured(true)
    CTInterceptorBuilder()
    CertificatePinner.Builder().add("example.com", "sha256/AAA=").build()
}
